audio-worklet/src/synthesizer.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+class Synthesizer {
+  /**
+   * Synthesizer manages the life cycle of voices and binds GUI input
+   * to audio params.
+   * @param {BaseAudioContext} context the audio context
```

Comment:
AudioContext, not BaseAudioContext.  You can't create a BaseAudioContext, and it seems unlikely that you'll use an OfflineAudioContext here. (Or are you going to allow an OfflineAudiocontext?)